spec/std/file_spec.cr

```diff
@@ -648,6 +648,59 @@ describe "File" do
     end
   end
 
+  describe "append" do
+    it "can create a new file" do
+      filename = "#{__DIR__}/data/temp_append.txt"
+      begin
+        File.append(filename, "hello")
+        File.read(filename).should eq("hello")
+      ensure
+        File.delete(filename)
+      end
+    end
+
+    it "can append to an extant file" do
```

Comment:
`extant` => `existing` ?

Reply:
Oh, didn't know they are the same! `existing` is clearer to me, but maybe it's just that I don't know my vocabulary ^^